05/app.js

```diff
@@ -4,41 +4,41 @@ console.log('DOM');
 const curr = document.querySelector('.js-curr');
 //Utwórz kolejny przycisk, który będzie rodzeństwem (bratem) dla wyszukanego w zmiennej curr elementu. Element ten niech zawiera napis usuń z koszyka.
 if (curr) {
+    const sectionElement = curr.parentElement;
     const newButton = document.createElement('button');
-    newButton.innerText = 'usuń z koszyka';
-    curr.parentElement.appendChild(newButton);
-
+    newButton.textContent = 'Usuń z koszyka';
+    sectionElement.appendChild(newButton);
+    console.log(newButton);
 
 // dla wszystkich elementów, które są rodzeństwem dla elementu o klasie .js-curr dodaj klasę .siblings (wykorzystaj pętlę)
 
-if(curr) {
-    const parent = curr.parentElement.children;
-
-    for (let i = 0; i < parent.length; i++) {
-    curr.parentElement.children[i].classList.add('.sibling');
-};
-    console.log(curr.parentElement.children[i]);
-}
+const jsParent = document.querySelectorAll('.articles__item');
+const allArticle = [];
 
+allArticle.forEach(function (element) {
```

Comment:
To pusta tablica więc `forEach()` nie wykona się ani razu.
Rozwiązanie wcześniejsze było dobrym pomysłem.